nuget/Microsoft.ML.OnnxRuntimeGenAI.nuspec

```diff
@@ -17,15 +17,15 @@
     <dependencies>
       <group targetFramework="NETSTANDARD" >
         <dependency id="Microsoft.ML.OnnxRuntimeGenAI.Managed" version="$version$" />
-        <dependency id="Microsoft.ML.OnnxRuntime$ext$" version="1.17.1" />
+        <dependency id="Microsoft.ML.OnnxRuntime$ort_nuget_ext$" version="$ort_version$" />
```

Comment:
I think you forgot to update the `$ext$` to `$genai_nuget_ext$` in the `id` on line 6.